src/lava/magma/core/process/ports/ports.py

```diff
@@ -181,9 +181,9 @@ def flatten(self) -> "ReshapePort":
         return self.reshape((self.size,))
```

Comment:
Regarding line 145-146:
That sounds as if you actually cycle through the whole connectivity graph. I.e., if port a is connected to port b of Process B. and Process B via ports b2->c, then I would expect port c to be returned as well according to the docstring. But I guess that's not what is meant here?

Reply:
see above